packages/plugins/src/data/utils.ts

```diff
@@ -19,12 +11,11 @@ export function getLocation(sourceFile: SourceFile, start: number, length: numbe
     start + length - 1
   );
 
-  //bump 0 to 1, so on and so forth, so that line and column can be correctly displayed in sarif.
   return {
-    startLine: startLine + 1,
-    startColumn: startColumn + 1,
-    endLine: endLine + 1,
-    endColumn: endColumn + 1,
+    startLine: startLine + INDEX_BUMP + COMMENT_BUMP,
+    startColumn: startColumn + INDEX_BUMP,
+    endLine: endLine + INDEX_BUMP + COMMENT_BUMP,
```

Comment:
I noticed in the attached images that red squiggly seems to be off. Something is likely off in this `getLocation` function

Reply:
Fixed. End column reported by Typescript was erroneously assumed to be the last affected character of the diagnostic node, however, the end column reported is the last affected character + 1. Please see new screen shot.